src/components/TopBar/Wallet/index.tsx

```diff
@@ -107,7 +93,7 @@ export function Wallet(props: { open?: boolean; component?: string }) {
         anchor="right"
         open={props.open ? true : false}
         onOpen={() => null}
-        onClose={() => history.push("/stake")}
+        onClose={() => history.goBack()}
```

Comment:
the one thing I hate about this is goBack will take you out of the app if your first entry point is into the wallet screen. i.e. if you go from https://google.com to https://app.olympusdao.finance/#/wallet, goBack will take you back to google.